plugins/modules/azure_rm_privateendpointdnszonegroup_info.py

```diff
@@ -0,0 +1,249 @@
+                            description:
+                                - Fully qualified domain name of the record.
+                            returned: always
+                            type: str
+                            sample: myPostgreSqlSrv-123.privatelink.postgres.database.azure.com
+                        ip_addresses:
+                            description:
+                                - IP addresses for the record.
+                            returned: always
+                            type: list
+                            sample: ['10.1.0.4']
+                        provisioning_state:
+                            description:
+                                - Provisioning state of the resource.
+                            returned: always
+                            type: str
+                            sample: Succeeded
+                        record_set_name:
+                            description:
+                                - Name of the record.
+                            returned: always
+                            type: str
+                            sample: myPostgreSqlSrv-123
+                        record_type:
+                            description:
+                                - Type of record.
+                            returned: always
+                            type: str
+                            sample: A
+                        ttl:
+                            description:
+                                - Time to live value of the record.
+                            returned: always
+                            type: int
+                            sample: 10
+        provisioning_state:
+            description:
+                - Provisioning state of the resource.
+            returned: always
+            type: str
+            sample: Succeeded
+'''
+
+try:
+    from msrestazure.azure_exceptions import CloudError
+except Exception:
+    # This is handled in azure_rm_common
+    pass
+
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
+
+
+class AzureRMPrivateEndpointDnsZoneGroupInfo(AzureRMModuleBase):
+
+    def __init__(self):
+
+        self.module_arg_spec = dict(
+            name=dict(type="str"),
+            private_endpoint=dict(type="str", required=True),
+            resource_group=dict(type="str", required=True),
+        )
+
+        self.results = dict(
+            changed=False,
+            groups=[],
+        )
+
+        self.name = None
+        self.private_endpoint = None
+        self.resource_group = None
+        self.results = dict(
+            changed=False,
+        )
+
+        super(AzureRMPrivateEndpointDnsZoneGroupInfo, self).__init__(self.module_arg_spec,
+                                                                     supports_check_mode=True,
+                                                                     supports_tags=False,
```

Comment:
Since the document inherits tags, it is recommended that argc be set to true and the tags parameter can be passed. The resources of the specified tags can be filtered through tags. Thank you very much!
```suggestion
                                                                     supports_tags=True,
```